sympy/functions/elementary/complexes.py

```diff
@@ -754,6 +852,43 @@ def _eval_is_algebraic(self):
 class transpose(Function):
     """
     Linear map transposition.
+
+    Examples
+    ========
+
+    >>> from sympy.functions import transpose
+    >>> from sympy.matrices import MatrixSymbol
+    >>> from sympy import Matrix
+    >>> A = MatrixSymbol('A', 25, 9)
+    >>> transpose(A)
+    A.T
+    >>> B = MatrixSymbol('B', 9, 22)
+    >>> transpose(B)
+    B.T
+    >>> transpose(A*B)
+    B.T*A.T
+    >>> Matrix([[4, 5], [2, 1], [90, 12]])
```

Comment:
Maybe do 

```py
>>> M = Matrix([[4, 5], [2, 1], [90, 12]])
>>> M
>>> transpose(M)
```

to make it clear it is the same matrix on both lines.

Reply:
Done.